person.rb

```diff
@@ -20,6 +22,10 @@ def correct_name
     @name
   end
 
+  def add_rental(date, book)
+    Rental.new(date, self, book)
```

Comment:
- [ ] Rental class initialized method is accepting `date, book, person` as parameters to create Rental, On this method while creating rental you are passing `date, person, book`. It will through an error if you create a rental using this function. Please fix this bug. You can consider using this code `Rental.new(date, book, self)`